style.css

```diff
@@ -6,23 +6,25 @@
 .container {
   background-color: #f7f7f9;
   width: 100%;
-  height: 4730px;
+  height: 6500px;
   margin: 0 auto;
   display: flex;
   flex-direction: column;
   align-items: center;
 }
 
 .introduction {
-  width: auto;
-  height: 720px;
+  width: 23.5rem;
+  height: 735px;
   background: #fff;
   margin-top: 10px;
   border-bottom-left-radius: 100px;
   margin-bottom: 80px;
   background-image: url(./images/background.png);
   background-repeat: no-repeat;
   background-position-y: 100px;
+
+  /* flex: 1; */
```

Comment:
- [ ] Please remove all commented code from your codebase to make it clean and easy to maintain